packages/netlify-cms-core/src/lib/__tests__/formatters.spec.js

```diff
@@ -113,6 +113,50 @@ describe('formatters', () => {
       ).toEqual('Custom commit message');
     });
 
+    it('should use empty values if "authorLogin" and "authorName" are missing in commit message', () => {
+      config.getIn.mockReturnValueOnce(
+        Map({
+          update: '{{author-login}} - {{author-name}}: Create {{collection}} “{{slug}}”',
+        }),
+      );
+
+      expect(
+        commitMessageFormatter(
+          'update',
+          config,
+          {
+            slug: 'doc-slug',
+            path: 'file-path',
+            collection,
+          },
+          true,
+        ),
+      ).toEqual(' - : Create Collection “doc-slug”');
+    });
+
+    it('should return custom update message with author information', () => {
```

Comment:
Just a minor issue with the test name. I tried fixing it myself but I can't push to your branch.
```suggestion
    it('should return custom create message with author information', () => {
```